source/eventcore/internal/corefoundation.d

```diff
@@ -1,6 +1,12 @@
 module eventcore.internal.corefoundation;
 
-version (darwin):
+version (OSX) enum isAppleOS = true;
+else version (iOS) enum isAppleOS = true;
+else version (TVOS) enum isAppleOS = true;
+else version (WatchOS) enum isAppleOS = true;
+else enum isAppleOS = false;
```

Comment:
What's the difference between `version (darwin)` and `isAppleOS` ?

Reply:
It might be that this has changed since, but my last state of understanding was that instead of generalizing `darwin` to what it should be, it was decided to keep it equivalent to `OSX` and deprecate it. I may have missed some later development, but the [docs](https://dlang.org/spec/version.html#predefined-versions) also still list it that way.